### PR TITLE
Serve single objects as well as collections.

### PR DESCRIPTION
**Pull request by btyoung**
Implemented by the addition of a ``Single`` class to complement the ``Collection`` class.

This is necessary to implement tests for proposed updates to ``ng-admin``

### CODE REVIEW

File: test/src/Server-spec.js [hunk 1]
```diff
@@ -24,12 +25,15 @@
         describe('init', function() {
             it('should populate several collections', function() {
                 var server = new Server();
+                console.log(server);
```
Comment (fzaninotto) [inline test/src/Server-spec.js]: please remove

**Comment by fzaninotto** [comment]
Could you also add documentation for the new feature in the README?

**Comment by btyoung** [comment]
No problem, I've removed the log call and added documentation.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**